### PR TITLE
Feature/tabs ui

### PR DESCRIPTION
**Pull request by vyvil**
This PR inluces updates on Tab navigation UI to follow newest wallet design.
Here's the new [design](https://app.zeplin.io/project/5b1535f192d75db2752e5adb/screen/5c618a736eccb77559a74e70).
With an update on "unread" badge placement:

unreads

Changes:
A react-native-image-capinsets is added to create active tab shadows on Android using 9 patch image.
Tabs.js component is updated to accept icons as png.

Task [164139593](https://www.pivotaltracker.com/story/show/164139593).

Screenshots:
tab_shadow_1
tab_shadow_2
tab_shadow_3
unreads_1
unreads_2

### CODE REVIEW

File: src/components/Tabs/Tabs.js [hunk 1]
```diff
@@ -18,109 +18,257 @@
...
+const TabOuterWrapper = styled.View`
+`;
+
+const TabWrapper = styled.View`
   flex-direction: row;
+  background-color: ${baseColors.pattensBlue};
+  padding: ${Platform.select({
+    ios: '2px',
+    android: '2px 0',
+  })}
```
Comment (jsidorenko) [inline src/components/Tabs/Tabs.js]: semicolon ;)

File: src/components/Tabs/Tabs.js [hunk 2]
```diff
@@ -18,109 +18,257 @@
...
-  border-style: solid;
+const TextWrapper = styled.View`
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  height: 32px;
+  margin-top: ${Platform.select({
+    ios: '0',
+    android: '3px',
+  })}
```
Comment (jsidorenko) [inline src/components/Tabs/Tabs.js]: semicolon :)

File: src/components/Tabs/Tabs.js [hunk 3]
```diff
@@ -18,109 +18,257 @@
...
+const UnreadBadge = styled.View`
+  height: 24px;
+  width: 24px;
+  border-radius: 12px;
+  background-color: ${baseColors.electricBlue};
+  position: absolute;
+  top: ${Platform.select({
+    ios: '4px',
+    android: '4.5px',
+  })}
```
Comment (jsidorenko) [inline src/components/Tabs/Tabs.js]: again

File: src/components/Tabs/Tabs.js [hunk 4]
```diff
@@ -18,109 +18,257 @@
...
+      tabImageNormal,
+      tabImageActive,
+      id,
+    } = tab;
+    const { activeTab } = this.state;
+    const isActive = activeTab === id;
+    const tabImage = isActive ? tabImageActive : tabImageNormal;
+
+    if (icon || tabImageActive || tabImageNormal) {
+      if (icon) {
```
Comment (jsidorenko) [inline src/components/Tabs/Tabs.js]: Let's put this above the `if (icon || tabImageActive || tabImageNormal) {`

File: android/rn-libs/react-native-image-capinsets/ImageCapInset.android.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  View,
+  Image,
+  requireNativeComponent,
+} from 'react-native';
+import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
+
+class ImageCapInset extends Component {
```
[the remark this answers is not in the document]
Comment (vyvil) [reply android/rn-libs/react-native-image-capinsets/ImageCapInset.android.js]: @diablourbano,
This is a source code of manually included library (to make it Android SDK compliant).
I'd rather not alter it's code.

File: android/rn-libs/react-native-image-capinsets/ImageCapInset.ios.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import {
+  ImageBackground,
+  Image,
+} from 'react-native';
+
+class ImageCapInset extends Component {
```
[the remark this answers is not in the document]
Comment (vyvil) [reply android/rn-libs/react-native-image-capinsets/ImageCapInset.ios.js]: same here :)

File: src/components/Tabs/Tabs.js (hunk 1 above)
Comment (vyvil) [reply src/components/Tabs/Tabs.js]: Will add missing ones :))

Comment (jsidorenko) [reply src/components/Tabs/Tabs.js]: Thanks, Ros would be happy :DD

File: src/components/Tabs/Tabs.js (hunk 4 above)
Comment (vyvil) [reply src/components/Tabs/Tabs.js]: Done :)

File: android/rn-libs/react-native-image-capinsets/ImageCapInset.android.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (diablourbano) [reply android/rn-libs/react-native-image-capinsets/ImageCapInset.android.js]: sure, that make sense

**Pull request merged**